src/authModules/keycloakAddress.ts

```diff
@@ -0,0 +1,44 @@
+import { Response } from 'express'
+import fetch from 'cross-fetch'
+import checkAbilities from '../utils/checkAbilities'
+interface keycloakResponse {
+  id: string
+  userRoles: [string]
+}
+
+export default async function address(
+  res: Response,
+  address: string,
+  eventType: string,
+  component: string
+): Promise<void> {
+  try {
+    const url = `${process.env.KEYCLOAK_ADDRESS_URL}/${address}`
+    const response = await fetch(url, {
+      method: 'GET'
+    })
+    const resJSON: [keycloakResponse] = await response.json()
+    // Respond flase if no users are found
+    if (resJSON.length < 1) {
+      res.json(false)
+    }
+    // Looping through all users that are returned for the address
+    for (let i = 0; i < resJSON.length; i++) {
+      const roles: [string] = resJSON[i].userRoles
+      let result = false
```

Comment:
Suggest to move `let result = false`  and  `res.json(result)` outside of the for loop. 
Currently will response the request on first iteration (for loop).

Will also encounter error when response multiple time to the client.